apps/frontend-pwa/src/pages/course/[courseId]/docs/index.tsx

```diff
@@ -54,7 +54,7 @@ export async function getServerSideProps(ctx: GetServerSidePropsContext) {
     ctx,
   })
 
-  if (participant && !participant.avatar) {
+  if (participant && !participant.isActive) {
```

Comment:
new field on participant that is set when the first save on the edit profile page occurs. this ensures that accounts are only active after the participant has seen this page at least once. reason: the page includes the data privacy disclaimer, which needs to be seen and acknowledged, before we can start tracking data. this also works for existing users, as we set the default to false, so they will see the page on first visit of the klicker in a new course.